cmd/utilities/go.mod

```diff
@@ -0,0 +1,48 @@
+// Copyright Avast Software. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+module github.com/trustbloc/wallet-sdk/cmd/utilities
```

Comment:
I think we should directly name this `gomobilewrappers` - it's a little bit more specific of a name

Reply:
I named it "utilities"  because it is probable that we will have other utility functionality, not only gomobilewrappers

Reply:
so we can put it there. Also putting gomobilewrappers into utilities also shows that it is not some independent functionality but just some utilities for wallet-sdk-gomobile

Reply:
Seems OK to me in that case - the only other things I can think of are to have multiple go mod files (one for each type of utility), or, move them back into `wallet-sdk-gomobile` but then update the gomobile build scripts to ignore the wrappers folder... but as discussed that will take some time. Maybe in the future as the project grows we can see if a different folder structure (for everything) would make more sense

Reply:
I think it's OK to merge unless anyone else has any ideas

Reply:
Probably it would be better to add a prefix which best describes the module to this rather having a generic name `utilities` @vkubiv. We can have a follow up PR.